token/context_test.go

```diff
@@ -1,107 +0,0 @@
-
-	serviceName := "test-service"
-
-	t.Run("Is Service Account", func(t *testing.T) {
-
-		t.Run("Valid", func(t *testing.T) {
-			// given
-			claims := jwt.MapClaims{}
-			claims["service_accountname"] = serviceName
-			ctx := goajwt.WithJWT(context.Background(), jwt.NewWithClaims(jwt.SigningMethodRS512, claims))
-			// then
-			assert.True(t, token.IsServiceAccount(ctx))
-		})
-
-		t.Run("Missing name", func(t *testing.T) {
-			// given
-			claims := jwt.MapClaims{}
-			ctx := goajwt.WithJWT(context.Background(), jwt.NewWithClaims(jwt.SigningMethodRS512, claims))
-			// then
-			assert.False(t, token.IsServiceAccount(ctx))
-		})
-
-		t.Run("Missing token", func(t *testing.T) {
-			// given
-			ctx := context.Background()
-			// then
-			assert.False(t, token.IsServiceAccount(ctx))
-		})
-
-		t.Run("Nil token", func(t *testing.T) {
-			// given
-			ctx := goajwt.WithJWT(context.Background(), nil)
-			// then
-			assert.False(t, token.IsServiceAccount(ctx))
-		})
-
-		t.Run("Wrong data type", func(t *testing.T) {
-			// given
-			claims := jwt.MapClaims{}
-			claims["service_accountname"] = 100
-			ctx := goajwt.WithJWT(context.Background(), jwt.NewWithClaims(jwt.SigningMethodRS512, claims))
-			// then
-			assert.False(t, token.IsServiceAccount(ctx))
-		})
-	})
-
-	t.Run("Is Specific Service Account", func(t *testing.T) {
-
-		t.Run("Valid", func(t *testing.T) {
-			// given
-			claims := jwt.MapClaims{}
-			claims["service_accountname"] = serviceName
-			ctx := goajwt.WithJWT(context.Background(), jwt.NewWithClaims(jwt.SigningMethodRS512, claims))
-			// then
-			assert.True(t, token.IsSpecificServiceAccount(ctx, "dummy-service", serviceName))
-		})
-
-		t.Run("Missing name", func(t *testing.T) {
-			// given
-			claims := jwt.MapClaims{}
-			ctx := goajwt.WithJWT(context.Background(), jwt.NewWithClaims(jwt.SigningMethodRS512, claims))
-			// then
-			assert.False(t, token.IsSpecificServiceAccount(ctx, serviceName))
-		})
-		t.Run("Nil token", func(t *testing.T) {
-			// given
-			ctx := goajwt.WithJWT(context.Background(), nil)
-			// then
-			assert.False(t, token.IsSpecificServiceAccount(ctx, serviceName))
-		})
-		t.Run("Wrong data type", func(t *testing.T) {
-			// given
-			claims := jwt.MapClaims{}
-			claims["service_accountname"] = 100
-			ctx := goajwt.WithJWT(context.Background(), jwt.NewWithClaims(jwt.SigningMethodRS512, claims))
-			// then
-			assert.False(t, token.IsSpecificServiceAccount(ctx, serviceName))
-		})
-		t.Run("Missing token", func(t *testing.T) {
-			// given
-			ctx := context.Background()
-			// then
-			assert.False(t, token.IsSpecificServiceAccount(ctx, serviceName))
-		})
-		t.Run("Wrong name", func(t *testing.T) {
-			// given
-			claims := jwt.MapClaims{}
-			claims["service_accountname"] = serviceName + "_asdsa"
-			ctx := goajwt.WithJWT(context.Background(), jwt.NewWithClaims(jwt.SigningMethodRS512, claims))
-			// then
-			assert.False(t, token.IsSpecificServiceAccount(ctx, serviceName))
-		})
-	})
-}
```

Comment:
I guess, these test cases are not added back ??

Reply:
good point. Let me add them back, indeed. I guess I remove them by mistake when I updated this pkg by importing from another repo :/